apps/inviwo/inviwo.cpp

```diff
@@ -55,6 +55,7 @@ int main(int argc, char** argv) {
     inviwo::LogCentral::init(&logger);
     auto logCounter = std::make_shared<inviwo::LogErrorCounter>();
     logger.registerLogger(logCounter);
+    qputenv("QT_STYLE_OVERRIDE","");
```

Comment:
I would add the link/comment here also, Should this only happen on Linux?

Reply:
As far as I can see, this problem is related to GTK which is a widget toolkit used by several desktop environments for Linux-based systems. It is primarily targeted for X11 and Wayland but can technically run on macOS and Windows. I think that the latter is a virtually non-existent use case though. So yes, I believe this should only happen on Linux. Should I add the corresponding macro as well?